### PR TITLE
Information and an sample for form and fieldset.

### PR DESCRIPTION
**Pull request by manngo**
I don’t know how to include the sample result, nor how to update the compatibility table. According to https://html.spec.whatwg.org/multipage/semantics-other.html#selector-invalid the `fieldset:invalid` is part of the spec. https://caniuse.com/?search=%3Ainvalid doesn’t mention `fieldset` specifically, but does include `form:invalid`; IE is, of course, out, while curent Edge is in (older Edge was out).

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by wbamberg** [comment]
There is a slightly tricky error here.

This page originally contained a live sample, which you can see from the call to the `{{EmbedLiveSample}}` macro. This macro takes a parameter that should be the ID (or alternatively, just the text content) of a heading in the page, and it then scoops up all the code blocks under that heading and sticks them in an iframe to show the result.

So before your change the relevant bit of doc structure was like this:

```
## Examples

### A form that colors elements green when they validate and red when they don't.

#### HTML

...

#### CSS

...

#### Result

{{EmbedLiveSample('Examples', 600, 120)}}
```

So the `{{EmbedLiveSample}}` call pointed to the "Examples" H2. It worked because there was only one example. Now you've added a new H3 "A form that shows sections in stages", and so all the code blocks in there are also being included in the output iframe for the first example, which doesn't make sense.

So what we have to do is

1. adjust the argument to the first `{{EmbedLiveSample}}` call so it is referencing the H3 "A form that colors elements green when they validate and red when they don't."
2. add a new `{{EmbedLiveSample}}` call that references your new example H3

Like this:

```
## Examples

### A form that colors elements green when they validate and red when they don't.

#### HTML

...

#### CSS

...

#### Result

{{EmbedLiveSample('A form that colors elements green when they validate and red when they don't.
', 600, 120)}}

### A form that shows sections in stages

#### HTML

...

#### CSS

...

#### Result

{{EmbedLiveSample('A form that shows sections in stages', 600, 120)}}

```

Having said that, "A form that colors elements green when they validate and red when they don't." is a terrible heading, and we should probably change it to something more like "Coloring elements to show validation".

Finally, we should increase the height of the result iframe in both examples, so the result fits properly without a scrollbar. You do this by changing the third parameter, which is currently 120 (pixels). I reckon 140 ought to work.

I hope this all makes sense! I might have some more review comments after, but this ought to get the examples working.

> how to update the compatibility table

That needs to get done over in the https://github.com/mdn/browser-compat-data repo. Specifically you'll need to adjust the JSON in https://github.com/mdn/browser-compat-data/blob/main/css/selectors/invalid.json, which is where the data lives that powers the table.

**Comment by wbamberg** [comment]
@manngo , did that make sense^ ? Let me now if you'd like me to push the relevant updates to this PR.

**Comment by wbamberg** [comment]
I just pushed an update to make the live samples work. Since I've substantially edited this now I should not review it. @teoli2003 would you mind reviewing?

File: files/en-us/web/css/_colon_invalid/index.md [hunk 1]
```diff
@@ -89,7 +89,70 @@ input:required:invalid {
 
 #### Result
 
-{{EmbedLiveSample('Examples', 600, 120)}}
+{{EmbedLiveSample('Coloring elements to show validation', 600, 200)}}
+
+### A form that shows sections in stages
```
Comment (teoli2003) [inline files/en-us/web/css/_colon_invalid/index.md:94]: NIT: what about:
```suggestion
### Showing sections in stages
```

File: files/en-us/web/css/_colon_invalid/index.md [hunk 2]
```diff
@@ -89,7 +89,70 @@ input:required:invalid {
...
+  border: thin solid darkgrey;
+  font-size: 1.25em;
+  background-color: darkgrey;
+  color: white;
+}
+```
+
+#### Result
+
+{{EmbedLiveSample('A form that shows sections in stages', 600, 300)}}
```
Comment (teoli2003) [inline files/en-us/web/css/_colon_invalid/index.md:155]: To keep coherence with the previous suggestion:
```suggestion
{{EmbedLiveSample('Showing sections in stages', 600, 300)}}
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
I proposed an alternative title for the second example. Else looks good.

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks @manngo and @wbamberg !

**Pull request merged**